crates/agglayer-types/src/lib.rs

```diff
@@ -219,10 +222,11 @@ pub struct Certificate {
     pub bridge_exits: Vec<BridgeExit>,
     /// List of imported bridge exits included in this state transition.
     pub imported_bridge_exits: Vec<ImportedBridgeExit>,
-    /// Signature committed to the bridge exits and imported bridge exits.
-    pub signature: Signature,
     /// Fixed size field of arbitrary data for the chain needs.
     pub metadata: Metadata,
+    /// Aggchain proof which is either one ECDSA or one SP1 plonk proof.
```

Comment:
```suggestion
    /// Aggchain proof which is either one ECDSA or one SP1 stark proof.
```

Reply:
I changed it to just `SP1 proof`. Whether it is plonk or stark or whatnot could be considered an implementation detail. Let me know if you disagree @hadjiszs.